actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -204,7 +200,6 @@ pub fn call_generic<RT: Runtime>(
                         // We provide enough gas for the transfer to succeed in all case.
                         gas = TRANSFER_GAS_LIMIT;
                     }
-                    let gas_limit = Some(effective_gas_limit(system, gas));
```

Comment:
I moved this down to be a bit more accurate (it applies the 63/64 rule closer to the actual call).